docs/proposed-api.md

```diff
@@ -119,14 +123,16 @@ metadata:
 spec:
   peering:
     vpc-1:
-      ips:
-        - cidr: 10.1.1.0/24
-      as:
-        - cidr: 1.2.3.0/24
+      expose:
+        - ips:
+            - cidr: 10.1.1.0/24
+          as:
+            - cidr: 1.2.3.0/24
```

Comment:
... Although this case does have `cidr:` for `as:`, apparently the only occurrence in the PR. Is this a mistake, or am I missing something?

Reply:
Yup, missed it